cypress/page/payment-step.page.ts

```diff
@@ -0,0 +1,27 @@
+class PaymentStepPage {
+    private bankWire: string;
+    private confirOrder: string;
+    private storeIsCompleted: string;
+
+    constructor() {
+        this.bankWire = ".bankwire";
+        this.confirOrder = "#cart_navigation > button > span"
```

Comment:
When we use `>`, we are specifying that the element on the left side is the parent of the element on the right side, this approach, sometimes is problematic because we are coupling the selector to the structure of the HTML, an in your example we have to specify all the path from the `#cart_navigation` to the `span` tag, which contains the confirm order button. However, if we just left one space, we are indicating that the element on the right side is inside of the element on the left side, which in most cases, will work in the same way with the advantage of not specifying all the path. So probably: `"#cart_navigation span` will be a better selector.

Look at [this page](https://www.w3schools.com/cssref/css_selectors.asp) as reference for CSS selectors